### ISSUE TITLE
Documentation of "How to use Tensorboard?": `with_logger` not defined

### ISSUE THREAD

**Issue by JinraeKim** [JuliaReinforcementLearning/ReinforcementLearning.jl#223]
I tried to follow [an example](https://juliareinforcementlearning.org/guide/#how_to_use_tensorboard) but I encountered an error, `with_logger` not defined.

I think it should be explained where it is defined or how to use it in the tutorial.

EDIT: Thank you all who have contributed for this project. I think this has some potentials to be the main tool for RL in Julia.
Unfortunately, I think that the documentation is bit hard to follow and understand how to implement this package. Probably the main reason is that the package is quite huge as the combination of multiple sub-packages for reuse.
In my opinion, detailed examples without errors would be very helpful for newbies like me.

**Comment by findmyway** [comment]
You need to run `using Logging` first.

Things evolve very fast before v0.8, so the documentation is not updated very frequently. But I'll work on evaluating benchmarks and updating docs in the next month before adding new algorithms.

**Comment by findmyway** [comment]
Also, thanks for putting forward the issues with documentation. Your feedback is very important to the usability of this package.
